### PR TITLE
Compliancy to guidelines regarding Source Control Management

### PR DESCRIPTION
**Pull request by markus-ap**
This PR adds the `SECURITY.md` file and updates the `README.md` file to stay compliant with relevant guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,25 @@ Description, schema and libraries for the "record" format for exchanging RDF
 * [A generator of very simple records](src/RecordGenerator)
 * [Queries and code for working with records in fuseki](src/fuseki)
 * [Setup, datalog and queries for working with records in RDFOx](src/rdfox)
+
+
+## CI/CD
+The CI/CD on this repository is done through Github Actions. The action [publish.yml](./.github/workflows/publish.yml) describes how it works.
```
Comment (daghovland) [inline README.md:13]: publish.yml fins ikke

File: README.md [hunk 2]
```diff
@@ -7,3 +7,25 @@ Description, schema and libraries for the "record" format for exchanging RDF
 * [A generator of very simple records](src/RecordGenerator)
 * [Queries and code for working with records in fuseki](src/fuseki)
 * [Setup, datalog and queries for working with records in RDFOx](src/rdfox)
+
+
+## CI/CD
```
Comment (daghovland) [inline README.md:12]:
```suggestion
## CI/CD For Records .NET package
```

Comment (daghovland) [reply README.md:12]: Dette gjelder jo bare for .NET-delen av repoet

**Review by daghovland — APPROVED** [review]
Dette blir jo bra.

File: README.md [hunk 3]
```diff
@@ -7,3 +7,25 @@ Description, schema and libraries for the "record" format for exchanging RDF
 * [A generator of very simple records](src/RecordGenerator)
 * [Queries and code for working with records in fuseki](src/fuseki)
 * [Setup, datalog and queries for working with records in RDFOx](src/rdfox)
+
+
+## CI/CD For Records .NET package
+The CI/CD on this repository is done through Github Actions. The action [publish.yml](./.github/workflows/publish.yml) describes how it works.
```
Comment (markus-ap) [inline README.md:13]:
```suggestion
The CI/CD on this repository is done through Github Actions. The action [publish.yml](./.github/workflows/publish.yaml) describes how it works.
```

**Review by daghovland — APPROVED** [review]

**Pull request merged**